src/api/common/plugins/swagger.js

```diff
@@ -0,0 +1,37 @@
+import HapiSwagger from 'hapi-swagger'
+import Inert from '@hapi/inert'
+import Vision from '@hapi/vision'
+
+import { config } from '~/src/config/index.js'
+
+/**
+ * Registers Swagger UI plugins (Inert, Vision, and HapiSwagger)
+ */
+const swagger = {
+  plugins: {
+    register: async (server) => {
+      await server.register([
+        Inert,
+        Vision,
+        {
+          plugin: HapiSwagger,
+          options: {
+            definitionPrefix: 'useLabel',
+            info: {
+              title: 'Land Grants API',
+              version: config.serviceVersion
+            }
```

Comment:
From security POV, might be worth only showing the docs in non-Prod environments
```suggestion
            },
            swaggerUI: !config.isProduction,
				    documentationPage: !config.isProduction,
```

Reply:
thanks @aliuk2012 makes sense